src/components/Display/Display.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Display from './Display';
+
+describe('Calculator', () => {
+    let wrapper;
```

Comment:
```suggestion
    let subject;
```
The usual name of the variable used as the unit being test is `subject`. This is just a convention of ours.